JenkinsfileStressTests

```diff
@@ -0,0 +1,83 @@
+def maxRetries = 120
+def waitForCommandSuccess(command, maxRetries) {
+    def tries = 0;
+    while (tries++ < maxRetries) {
+        try {
+            sh command
+            return true
+        } catch (Exception e) {
+            if (tries >= maxRetries) {
+                throw new Exception("${maxRetries} tries exceeded. ${command} failed.", e)
+            }
+            sh "sleep 1"
+        }
+    }
+}
+
+pipeline {
+    agent { label 'my-slave' }
+
+    tools {
+        maven 'maven'
+    }
+
+    stages {
+        stage('Compile') {
+            steps {
+                sh "mvn clean install -DskipTests"
+            }
+        }
+        stage('Build image') {
+            steps {
+                script {
+                    docker.withTool('docker') {
+                        sh "cp server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.jar dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                        sh "cp -r server/container/guice/cassandra-rabbitmq-guice/target/james-server-cassandra-rabbitmq-guice.lib dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                        sh "cp server/container/cli/target/james-server-cli.jar dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                        sh "cp -r server/container/cli/target/james-server-cli.lib dockerfiles/run/guice/cassandra-rabbitmq/destination"
+                        sh "docker build -t james_run dockerfiles/run/guice/cassandra-rabbitmq"
+                    }
+                }
+            }
+        }
+        stage('Start James') {
+            steps {
+                script {
+                    docker.withTool('docker') {
+                        sh "docker run -d --name=cassandra cassandra:3.11.3"
+                        sh "docker run -d --name=rabbitmq rabbitmq:3.7.7-management"
+                        sh "docker run -d --name=elasticsearch elasticsearch:2.4.6"
+                        try {
+                            waitForCommandSuccess("docker exec cassandra cqlsh -e 'describe columnfamily system.local;'", maxRetries)
+                        } catch (Exception e) {
+                            sh "docker logs cassandra"
+                        }
+
+                        sh "docker run -d --hostname HOSTNAME -p 25:25 -p 8081:80 -p 110:110 -p 143:143 -p 465:465 -p 587:587 -p 993:993 --link cassandra:cassandra --link rabbitmq:rabbitmq --link elasticsearch:elasticsearch --name james_run -t james_run"
```

Comment:
I don't see a link to swift

Reply:
It's not the product we wanted to stress yet; and may be never, as Swift will be replaced by S3.

BTW, you can modify this file on your own branch in order to check the another James product.